app.js

```diff
@@ -21,6 +22,7 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/', indexRouter, gamesRouter, studioRouter);
+app.use('/games', createRouter)
```

Comment:
What are your thoughts on having the games routing split between two route files?

Reply:
I don't necessarily have any thoughts but I could've gone farther and separated them by mutations and read. That way its sorta more organized.

Reply:
I would go ahead and separate them entirely. We want our developers to be able to go to one place to work on the games resource.

Reply:
Wait separate or combine them?